ffi/dart/tools/datagen/Cargo.toml

```diff
@@ -0,0 +1,49 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "icu_datagen_precompiled"
```

Comment:
It is Dart specific, it will only support Dart keys in the future. This is small enough to not generalise right now.

Reply:
I would prefer if the package were named as such. `icu_datagen_precompiled` is not a great name because:

1. Somewhat confusing with `--fallback preresolved` or `--features compiled_data` when this does neither of those things
2. Sounds like a general utility not specific to Dart

Reply:
`icu_datagen_dart`?

Reply:
Ok